### ISSUE TITLE
Bug with count in get_full_connections

### ISSUE THREAD

**Issue by Antonkoch999** [sns-sdks/python-facebook#264]
Replace this: 
`    def get_full_connections(
        self,
        object_id: str,
        connection: str,
        count: Optional[int] = 10,
        limit: Optional[int] = None,
        **kwargs,
    ) -> dict:
`    
to
`    def get_full_connections(
        self,
        object_id: str,
        connection: str,
        count: Optional[int] = None,
        limit: Optional[int] = None,
        **kwargs,
    ) -> dict:
`

**Comment by MerleLiuKun** [comment]
Yes, the count is limit the total count, If you want get all, just pass `count=None`.
